modules/customize-certificates-replace-default-router.adoc

```diff
@@ -24,6 +24,8 @@ wildcard certificate must be the first certificate in the file. It can then be
 followed with any intermediate certificates, and the file should end with the
 root CA certificate.
 * Copy the root CA certificate into an additional PEM format file.
+* Verify that all certificates which include `-----END CERTIFICATE-----` also
+end with one carriage return after that line.
```

Comment:
Sorry to nitpick over this one thing on the merge review, but could you combine this into one line in the adoc file? We're not supposed to hard wrap lines per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines):
```suggestion
* Verify that all certificates which include `-----END CERTIFICATE-----` also end with one carriage return after that line.
```

Reply:
found no hard wraps

Reply:
After troubleshooting with James, we couldn't find out why this sentence (and all other lines in this file) are getting hard wrapped despite appearing correctly in his editor (which has any hard wrap related settings turned off). Going to merge the PR as is and investigate some more after the fact.